lamp/create-group.php

```diff
@@ -72,5 +72,14 @@ function render_create_group_page($error=null) { ?>
             if (!validate_input($_POST, "group_name")) return render_create_group_page("Please provide a group name");
 
             $group_name = parse_input('group_name', true);
-            Channel::create_channel($course->course_id, $group_name, 2);
+            $group = Channel::create_channel($course->course_id, $group_name, 2);
+            GroupMembership::create_membership($_SESSION['uid'], $group->ch_id);
```

Comment:
Probably don't need to create a membership for the current user unless they requested it